qiskit/quantum_info/operators/scalar_op.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+ScalarOp class
+"""
+
+from numbers import Number
+import numpy as np
+
+from qiskit.exceptions import QiskitError
+from qiskit.quantum_info.operators.base_operator import BaseOperator
+from qiskit.quantum_info.operators.operator import Operator
+
+
+class ScalarOp(BaseOperator):
```

Comment:
shouldn't this just derive from Operator?

Reply:
No, then it would inherit its array fields and all the other unnecessary numerical baggage.